### ISSUE TITLE
Crash on dial trough ComputerCraft

### ISSUE THREAD

**Issue by Razor170** [Povstalec/StargateJourney#63]
Server crashes on dialing via ComputerCraft. Crash happens the moment the chevron gets engaged. Used Gate is Pegasus-Model.
Connection to ComputerCraft via BasicInterface -> WiredModem -> NetworkCable -> WiredModem -> Computer.
Crash also occures when replacing the BasicInterface with a CrystalInterface.

Edit: Server also keeps on Crashing after the program is executed because the Dial Sequence is loaded into the Gate and is being executed. The server crashes (assumption as it is not possible to establish a connection in time) on chevron engage.

Minecraft: Java 1.20.1
Used ModPack: ATM 9 - 0.2.37
Relevant Mods:

- Advanced Peripherals 0.7.35b
- CC:Tweaked Forge 1.109.3
- Stargate Journy 0.6.8 Hotfix

[crash-2024-01-20_09-19-05.txt](https://github.com/Povstalec/StargateJourney/files/13997473/crash-2024-01-20_09-19-05.txt)

TL;DR-CrashLog:

`java.lang.NullPointerException: Cannot invoke "net.povstalec.sgjourney.common.cctweaked.peripherals.BasicPeripheralWrapper.queueEvent(String, Object[])" because "this.peripheralWrapper" is null
	at net.povstalec.sgjourney.common.block_entities.BasicInterfaceEntity.queueEvent(BasicInterfaceEntity.java:187) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.povstalec.sgjourney.common.block_entities.stargate.AbstractStargateEntity.updateInterfaceBlocks(AbstractStargateEntity.java:756) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.povstalec.sgjourney.common.block_entities.stargate.AbstractStargateEntity.encodeChevron(AbstractStargateEntity.java:187) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.povstalec.sgjourney.common.block_entities.stargate.PegasusStargateEntity.encodeChevron(PegasusStargateEntity.java:119) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.povstalec.sgjourney.common.block_entities.stargate.PegasusStargateEntity.animateSpin(PegasusStargateEntity.java:166) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.povstalec.sgjourney.common.block_entities.stargate.PegasusStargateEntity.tick(PegasusStargateEntity.java:178) ~[Stargate%20Journey-1.20.1-0.6.8%20Hotfix.jar%23826!/:0.6.8] {re:classloading}
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:689) ~[server-1.20.1-20230612.114412-srg.jar%23895!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:782) ~[server-1.20.1-20230612.114412-srg.jar%23895!/:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.level.Level.redirect$bhk000$redirectTick(Level.java:3540) ~[server-1.20.1-20230612.114412-srg.jar%23895!/:?] `

**Comment by Povstalec** [comment]
I'm fairly sure I fixed in one of the versions after 0.6.8 (although ATM9 is still stuck on 0.6.8).

I don't remember why exactly the wrapper was null, but I think it was related to the Interface just not updating after being placed down or something similar.
